packages/sdc/components/03-organisms/campaign/campaign.css

```
@@ -32,25 +33,55 @@
 .ct-campaign .ct-campaign__inner > * {
-  margin-bottom: 1rem;
+  margin-bottom: 1.5rem;
+}
+.ct-campaign .ct-campaign__inner > .ct-campaign__title {
+  font-size: var(--ct-typography-heading-3-font-size);
+  line-height: var(--ct-typography-heading-3-line-height);
+  font-family: var(--ct-typography-heading-3-font-name);
+  font-weight: var(--ct-typography-heading-3-font-weight);
+  letter-spacing: var(--ct-typography-heading-3-letter-spacing);
 }
 @media (min-width: 768px) {
```

Stylelint (17.14.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)